### PR TITLE
Clarified menuitem shortcut property is for the label only

### PR DESCRIPTION
**Pull request by lostkeys**
Clarified menuitem shortcut property is for the label only. Also created additional demo to showcase adding a shortcut to a custom menuitem

Related Ticket: DOC-627

Description of Changes:
* Rewrote the table row for shortcut property on basic and nested menuitems to more accurately describe what happens.
* On the same page as above I removed the _shortcut modifier key mappings_ section as it wasn't relevant any more
* Added an example of using shortcuts with custom menuitems

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] ~~`_data/nav.yml` has been updated (if applicable)~~
- [x] Files has been included where required (if applicable)
- [x] ~~Files removed have been deleted, not just excluded from the build (if applicable)~~
- [x] ~~(New product features only) Release Note added~~

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _includes/live-demos/custom-shortcut-2/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            }
+          }
+        ];
+        callback(items);
+      }
+    });
+  }
+});
+
+function insertUsername() {
```
Comment (lnewson) [inline _includes/live-demos/custom-shortcut-2/index.js:29]: We don't normally suggest doing this, so this should probably be moved inside the setup block. eg:

```
setup: function (editor) {
  var insertUsername = function () {
    editor.insertContent(`@username`);
  }
}
```

Comment (lostkeys) [reply _includes/live-demos/custom-shortcut-2/index.js:29]: @lnewson You mean using `TinyMCE.activeEditor`? @spocke had a simiar remark for another thing, but our docs is littered with that string. 188 hits over 53 files....

Comment (lnewson) [reply _includes/live-demos/custom-shortcut-2/index.js]: Yeah, using `tinymce.activeEditor` can lead to issues in edge cases.

**Review by tylerkelly13 — APPROVED** [review]

**Review by lnewson — APPROVED** [review]

File: _includes/live-demos/custom-shortcut-2/index.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<textarea id="custom-shortcut-2">
+  <p>Use the shortcut <code>Cmd+Option+U</code> on Mac or <code>Ctrl+Alt+U</code> if your on Windows or press the plus button to insert a username here: </p>
```
Comment (metricjs) [inline _includes/live-demos/custom-shortcut-2/index.html:2]: your -> you are. Also commas maybe?

Comment (tylerkelly13) [reply _includes/live-demos/custom-shortcut-2/index.html]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1651/commits/1b33c5835a4e1b0494839449271cb103f5074764

**Review by metricjs — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Pull request merged**